Sources/Operators/ZipMany.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ZipMany.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 16/03/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+public extension Publisher {
+    // MARK: - Non-transform zipping variants.
+
+    /// Zips `self` with an array’s-worth of publishers with the same output and failure types.
+    /// Since there can be any number of `others`, `[Output]` values are emitted after zipping.
+    /// - Parameter others: The other publishers to zip with.
+    ///
+    /// - returns: A type-erased publisher with value events from each of the inner publishers zipped together in an array.
```

Comment:
```suggestion
    /// Zips `self` with an array of publishers with the same output and failure types.
    ///
    /// Since there can be any number of `others`, arrays of `Output` values are emitted after zipping.
    ///
    /// - parameter others: The other publishers to zip with.
    ///
    /// - returns: A type-erased publisher with value events from each of the inner publishers zipped together in an array.
```

Reply:
some minor formatting/word-tweaking, not sure why the diff makes it seem like I changed the whole thing :]